### PR TITLE
✨ [#1733] Add share on social media buttons for products

### PR DESCRIPTION
**Pull request by stevenbal**

### CODE REVIEW

File: src/open_inwoner/templates/pages/product/detail.html [hunk 1]
```diff
@@ -164,5 +164,15 @@ <h4 class="h4">{{ contact.first_name }} {{ contact.last_name }}</h4>
             {% endrender_column %}
         {% endrender_grid %}
 
+        <nav class="sharing-list" aria-label="{% trans "Deel dit artikel" %}">
+            <h3 class="h3" id="sharing">{% trans "Deel dit artikel" %}</h3>
+            {% with request.build_absolute_uri as page_url %}
+                {% include "components/ShareButton/ShareButton.html" with platform="facebook" platform_verbose="Facebook" text_to_share=page_url %}
```
Comment (stevenbal) [inline src/open_inwoner/templates/pages/product/detail.html:170]: Not sure if there are specific platforms missing here

**Comment by stevenbal** [comment]
Added the buttons to the bottom of the article: 
image

**Comment by alextreme** [comment]
@pi-sigma this PR from Steven looks good, but please add a single toggle in the Algemene Configuratie to determine if the sharing options are to be shown on the product pages or not (default can be on/True). I'm mostly thinking about the usecase of certain municipalities where they may not want to show social media options.

**Review by pi-sigma — APPROVED** [review]

**Comment by pi-sigma** [comment]
@alextreme Good to go, though you might take a look at the tests to check if I missed something.

**Review by alextreme — APPROVED** [review]
This looks good to me, thanks for wrapping this up Paul

**Pull request merged**